lib/actions.js

```diff
@@ -188,3 +188,13 @@ actions.addNullDataSourceItem = function() {
     value: null
   });
 };
+
+actions.templateFilter = function() {
+  if (this.options.loopbackVersion === '2.x') {
+    for (var template in this.templates) {
+      if (this.templates[template].name.indexOf('hello-world') > -1) {
```

Comment:
> After controller adding to hello-world template, it will not support 2.x, so I delete the choice of hello-world when 2.x is applied.

This does not make sense to me. The hello-world template implements a controller-like model that's fully compatible with LoopBack 2.x.